uiowa/uiowa
# Set Webform configuration on HTML Editor > Element text format to Filtered HTML

Sets the Element text format setting for all webforms under Webforms > Configuration > Elements > HTML editor settings to Filtered HTML. This should allow HTML or Markup fields to display text according to that text format. Fixes #3133 .
NOTE: Existing default contact forms are unaffected because they do not use the Markup field.

# How to test

- blt ds --site=socialwork.uiowa.edu
- Navigate to [http://socialwork.local.drupal.uiowa.edu/graduate/msw-social-work/iowa-city/information-sessions](http://socialwork.local.drupal.uiowa.edu/graduate/msw-social-work/iowa-city/information-sessions) or any of the other information session forms to verify that the HTML markup field at the top of the form is formatted correctly.
- Profit!
